qtbroker.py

```diff
@@ -13,6 +13,11 @@
 from matplotlib.backends import qt_compat
 
 
+class Placeholder:
```

Comment:
The export widget is added only after we have a header to look at. Until then, this (invisible) placeholder is put there. Since it has the same interface as the actual export widget, I don't need to special case our response to the *first* header loaded separately from all subsequent ones.